pydash/pydash_web/controller/dashboards.py

```diff
@@ -0,0 +1,45 @@
+"""
+Manages the lookup and returning of dashboard information for a certain user.
+
+Currently only returns static mock data.
+"""
+
+from flask import jsonify
+
+
+# Currently returns mock-data.
+def get_dashboard(dashboard_id):
```

Comment:
I'd suggest removing `get` from the name, because it does not give us more information.